pkg/yurthub/filter/approver.go

```diff
@@ -36,38 +36,35 @@ import (
 
 type approver struct {
 	sync.Mutex
-	reqKeyToName    map[string]string
-	configMapSynced cache.InformerSynced
-	stopCh          chan struct{}
+	reqKeyToName                       map[string]string
+	configMapSynced                    cache.InformerSynced
+	supportedResourceAndVerbsForFilter map[string]map[string]sets.String
+	defaultReqKeyToName                map[string]string
+	stopCh                             chan struct{}
 }
 
 var (
-	supportedVerbs           = sets.NewString("get", "list", "watch")
-	defaultWhiteListRequests = sets.NewString(reqKey(projectinfo.GetHubName(), "configmaps", "list"), reqKey(projectinfo.GetHubName(), "configmaps", "watch"))
-	defaultReqKeyToName      = map[string]string{
-		reqKey("kubelet", "services", "list"):                    MasterServiceFilterName,
-		reqKey("kubelet", "services", "watch"):                   MasterServiceFilterName,
-		reqKey("kube-proxy", "services", "list"):                 DiscardCloudServiceFilterName,
-		reqKey("kube-proxy", "services", "watch"):                DiscardCloudServiceFilterName,
-		reqKey("nginx-ingress-controller", "endpoints", "list"):  ServiceTopologyFilterName,
-		reqKey("nginx-ingress-controller", "endpoints", "watch"): ServiceTopologyFilterName,
-		reqKey("kube-proxy", "endpointslices", "list"):           ServiceTopologyFilterName,
-		reqKey("kube-proxy", "endpointslices", "watch"):          ServiceTopologyFilterName,
-		reqKey("coredns", "endpoints", "list"):                   ServiceTopologyFilterName,
-		reqKey("coredns", "endpoints", "watch"):                  ServiceTopologyFilterName,
-		reqKey("coredns", "endpointslices", "list"):              ServiceTopologyFilterName,
-		reqKey("coredns", "endpointslices", "watch"):             ServiceTopologyFilterName,
-	}
+	// defaultBlackListRequests is used for determining request to pass through filters.
+	defaultBlackListRequests = sets.NewString(reqKey(projectinfo.GetHubName(), "configmaps", "list"), reqKey(projectinfo.GetHubName(), "configmaps", "watch"))
```

Comment:
Does it mean that items in the `BlackList` will not be filtered? This comment seems to be ambiguous.

Reply:
comments is improved.